com.unity.robotics.urdf-importer/Runtime/MeshProcessing/StlAssetPostProcessor.cs

```diff
@@ -26,61 +26,69 @@ public class StlAssetPostProcessor : AssetPostprocessor
     public class StlAssetPostProcessor
 #endif    
     {
+        private static Material s_DefaultDiffuse = null;
+        
         static void OnPostprocessAllAssets(string[] importedAssets, string[] deletedAssets, string[] movedAssets, string[] movedFromPath)
         {
+            const bool enableStlPostprocessing = false; // AIRO-908 
```

Comment:
Is this bool being changed to true anywhere in the code ?

Reply:
I had it just in case we change our mind and decide to add the post-processing back. 
I just made a commit and removed the entire function altogether and added a comment instead.